test/getConfig.spec.js

```diff
@@ -211,6 +211,17 @@ describe('validateConfig', () => {
     expect(console.printHistory()).toMatchSnapshot()
   })
 
+  it('should not throw and should print validation warnings for old config', () => {
```

Comment:
Can we change this test title to `should print deprecation warning for gitDir option`?